File: C/Bronze/10798.c
// 10798번
#include <stdio.h>

int main(void){

    char words[5][15] = { 0 }; // 문자열 배열 선언

    for(int i = 0; i < 5; i++){
        scanf("%s", words[i], sizeof(words));
    }

    for(int j = 0; j < 15; j++){
        for(int i = 0; i < 5; i++){
            if(words[i][j] == NULL)
                continue;
            printf("%c", words[i][j]);
        }
    }

    return 0;
}

File: Python/Bronze/10798.py
words = [[0] * 15 for _ in range(5)]

for i in range(5):
    w = list(input())
    
    for j in range(len(w)):
        words[i][j] = w[j]

for j in range(15):
    for i in range(5):
        if words[i][j] == 0:
            continue
        print(words[i][j], end ='')
